documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -9,17 +9,11 @@
 
 toc::[]
 
-{prodname}'s Oracle Connector can monitor and record all of the row-level changes in the databases on an Oracle server.
-Most notably, the connector does not yet provide full-blown support of changes to the structure of captured tables (e.g. `ALTER TABLE...`) after the initial snapshot has been completed
-(see {jira-url}/browse/DBZ-718[DBZ-718]).
-It is supported though to capture tables newly added while the connector is running
-(provided the new table's name matches the connector's filter configuration).
-For table schema changes an xref:surrogate-schema-evolution[auxilliary solution] could be used.
-
 [[oracle-overview]]
 == Overview
 
 {prodname} ingests change events from Oracle using the native LogMiner database package or the https://docs.oracle.com/database/121/XSTRM/xstrm_intro.htm#XSTRM72647[XStream API].
+Oracle versions 11, 12, and 19 are supported.
```

Comment:
On https://debezium.io/releases/, we only mention 12 and 19?

Reply:
Ah that's right because we didn't list 11 since we cannot officially _test_ that version.  I've updated the PR accordingly.

Reply:
Thx!